portality/static/js/edges/public.journal.edge.js

```diff
@@ -4,8 +4,17 @@ $.extend(true, doaj, {
         activeEdges : {},
 
         embedSnippet : function(renderer) {
-            var snip = '<script type="text/javascript">!window.jQuery && document.write("<scr" + "ipt type=\"text/javascript\" src=\"http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js\"></scr" + "ipt>"); </script><script type="text/javascript">var doaj_url="https://doaj.org"; var SEARCH_CONFIGURED_OPTIONS={{QUERY}}</script><script src="https://doaj.org/static/widget/fixed_query.js" type="text/javascript"></script><div id="doaj-fixed-query-widget"></div></div>';
-            var query = renderer.component.edge.currentQuery.objectify({
+            var snip =
```

Comment:
This is change from https://github.com/DOAJ/doajPM/issues/2843 with jquery script removed